code/__DEFINES/MC.dm

```diff
@@ -1,3 +1,12 @@
+
+/**
+ * The Master Controller sleeps before running initialization logic
+ * allowing clients which were connected from a previous round to establish a connection
+ * and load in before it begins the process of initializing all other subsystems;
+ * thus ensuring that those clients do not drop the connection and need to manually reconnect.
+ */
```

Comment:
i know mothblocks told you to do this here, but this comment should be in the codedoc for master/Initialize, not here.